packages/interbit/src/tests/chainManagement/watchChain.test.js

```diff
@@ -0,0 +1,3 @@
+describe('watchChain', () => {
+  it('errors if the genesis blocks have changed when chains already exist')
+})
```

Comment:
I assume that this is just a placeholder for something you intend to do? Does this mean that the existing watcher tests (`packages/interbit/tests/chainManagement/watchChain.test.js`) no longer work?

Reply:
Correct, this is another piece from cascading deployment.